arrow-core-data/src/main/kotlin/arrow/core/NonEmptyList.kt

```diff
@@ -331,8 +331,9 @@ class NonEmptyList<out A>(
       NonEmptyList(head.combine(b.head), tail.salign(SA, b.tail).toList())
     }
 
-  fun <B> padZip(other: NonEmptyList<B>): NonEmptyList<Tuple2<A?, B?>> =
-    NonEmptyList(Tuple2(head, other.head), tail.padZip(other.tail))
+  @JvmName("padZipTupled")
```

Comment:
Why do we need this `@JvmName`?

Reply:
No, we don't need it. I think I put it when I was keeping the version of `padZip` with tuples, but we can get rid of it now.